### PR TITLE
Add console as a feature

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds the console interface as a feature.

### CODE REVIEW

File: features/console.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Console
+description: To write...
```
Comment (captainbrosset) [inline features/console.yml:2]:
```suggestion
description: The `console` API logs debugging messages to the browser development tools' console.
```

File: features/console.yml.dist [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "5"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "16"
+  #   firefox_android: "16"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.profileEnd_static
```
Comment (captainbrosset) [inline features/console.yml.dist:165]: BCD says that `profile` was removed in Firefox in 102. But BCD also says that `profileEnd` didn't get the same treatment, and that seems like a BCD bug to me. I would expect both to either be present or not.
In fact, both are defined for me in Firefox Nightly right now.

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: features/console.yml.dist (hunk 2 above)
Comment (captainbrosset) [reply features/console.yml.dist:165]: I see you've removed `profile` but left `profileEnd`. I guess we should instead either have them both in, or out. In this case, I think this is more likely a BCD bug. When I tested quickly, both functions seemed available in Firefox.

**Review by captainbrosset — APPROVED** [review]
I think this is good to merge as is, but we need to do something about profile/profileEnd, I'd suggest opening a BCD issue to investigate if the Firefox support data needs to be updated.

**Comment by Elchi3** [comment]
I opened https://github.com/mdn/browser-compat-data/pull/24675

**Pull request merged**